dotnetv3/S3/LifecycleExample/LifecycleExample/Lifecycle.cs

```diff
@@ -0,0 +1,202 @@
+                            Transitions = new List<LifecycleTransition>
+                            {
+                                new LifecycleTransition
+                                {
+                                    Days = 0,
+                                    StorageClass = S3StorageClass.Glacier,
+                                },
+                            },
+                        },
+                        new LifecycleRule
+                        {
+                            Id = "Archive and then delete rule",
+                            Filter = new LifecycleFilter()
+                            {
+                                LifecycleFilterPredicate = new LifecyclePrefixPredicate()
+                                {
+                                    Prefix = "projectdocs/",
+                                },
+                            },
+                            Status = LifecycleRuleStatus.Enabled,
+                            Transitions = new List<LifecycleTransition>
+                            {
+                                new LifecycleTransition
+                                {
+                                    Days = 30,
+                                    StorageClass = S3StorageClass.StandardInfrequentAccess,
+                                },
+                                new LifecycleTransition
+                                {
+                                    Days = 365,
+                                    StorageClass = S3StorageClass.Glacier,
+                                },
+                            },
+                            Expiration = new LifecycleRuleExpiration()
+                            {
+                                Days = 3650,
+                            },
+                        },
+                    },
+                };
+
+                // Add the configuration to the bucket.
+                await AddExampleLifecycleConfigAsync(client, bucketName, lifeCycleConfiguration);
+
+                // Retrieve the existing configuration to show that the configuration
+                // was added.
+                lifeCycleConfiguration = await RetrieveLifecycleConfigAsync(client, bucketName);
+
+                // Add another rule to the existing lifecycle rule.
+                lifeCycleConfiguration.Rules.Add(new LifecycleRule
+                {
+                    Id = "NewRule",
+                    Filter = new LifecycleFilter()
+                    {
+                        LifecycleFilterPredicate = new LifecyclePrefixPredicate()
+                        {
+                            Prefix = "YearlyDocuments/",
+                        },
+                    },
+                    Expiration = new LifecycleRuleExpiration()
+                    {
+                        Days = 3650,
+                    },
+                });
+
+                // Add the lifecycle configuration to the bucket.
```

Comment:
to the bucket -> to the Amazon S3 bucket

Reply:
Chamged to S3 bucket.